### ISSUE TITLE
chrome initiate two times toolbar.js

### ISSUE THREAD

**Issue by Bhavik-kumar** [magento/magento2#25934]
### Preconditions (*)
1. vanilla Magento CE 2.3.*
2. chrome browser
3. vanilla with sample data

### Steps to reproduce (*)
1. got to product list, search or advance search page
2. open **developer tools -> network(tab)**(Chrome browser)
3. Select/click any of the toolbar option like Grid/List, Sort By, or show X per page
4. **Network**(Chrome Browser) selected/clicked link triggering two time and because of two time trigger backed doing twice process after canceled one also.
(for verify put any log in to catalog->product->list controller).

![image](https://user-images.githubusercontent.com/7893647/70320003-aea5c480-1849-11ea-8e11-52e2e73911f9.png)

### Expected result (*)
1. Link should trigger once only or binding of  event make once with 
`vendor/magento/module-catalog/view/frontend/web/js/product/list/toolbar.js`

### Actual result (*)
1. On click/change link trigger two times 

### Solution

> File: vendor/magento/module-catalog/view/frontend/web/js/product/list/toolbar.js::_create(

```
 _create: function () {
	if(window.isToolbarLoaded)
		return;

	this._bind($(this.options.modeControl), this.options.mode, this.options.modeDefault);
	this._bind($(this.options.directionControl), this.options.direction, this.options.directionDefault);
	this._bind($(this.options.orderControl), this.options.order, this.options.orderDefault);
	this._bind($(this.options.limitControl), this.options.limit, this.options.limitDefault);

	window.isToolbarLoaded = true;
},
```

**Comment by Bhavik-kumar** [comment]
I am confirming that issue is able to reproduce with vanilla Magento instance.

**Comment by krishprakash** [comment]
@magento give me 2.4-develop instance

**Comment by magento-engcom-team** [comment]
Hi @krishprakash. Thank you for your request. I'm working on Magento 2.4-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @krishprakash, here is your [Magento instance](https://i-25934-2-4-develop.instances.magento-community.engineering).
Admin access: [https://i-25934-2-4-develop.instances.magento-community.engineering/admin](https://i-25934-2-4-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @krishprakash 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-29497` were created

**Issue Available**: @krishprakash, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by krishprakash** [comment]
Hi @Bhavik-kumar. Thank you for your report.
You can create PR with given solution in description

**Comment by krzksz** [comment]
@magento give me 2.4-develop instance

**Comment by magento-engcom-team** [comment]
Hi @krzksz. Thank you for your request. I'm working on Magento 2.4-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @krzksz, here is your [Magento instance](https://i-25934-2-4-develop.instances.magento-community.engineering).
Admin access: [https://i-25934-2-4-develop.instances.magento-community.engineering/admin](https://i-25934-2-4-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
Hi @Bhavik-kumar. Thank you for your report. 
The issue has been fixed in magento/magento2#28838 by @ptylek in 2.4-develop branch
Related commit(s):
 - [def4e04a3d28334a05e99926f8e88eb4f679439d](https://github.com/magento/magento2/commit/def4e04a3d28334a05e99926f8e88eb4f679439d)
 - [630bebd12e55264ac2ddd37fb4f1fe2d2a930903](https://github.com/magento/magento2/commit/630bebd12e55264ac2ddd37fb4f1fe2d2a930903)
 - [f5f8556ce6b1b7a050be1a1e5554f6b7f6919051](https://github.com/magento/magento2/commit/f5f8556ce6b1b7a050be1a1e5554f6b7f6919051)
 - [6cb1924666cf21c7d42b8a9052f5122c58cde721](https://github.com/magento/magento2/commit/6cb1924666cf21c7d42b8a9052f5122c58cde721)
 - [7efd88d1e7352f97fcc7723577c051bca2aa0831](https://github.com/magento/magento2/commit/7efd88d1e7352f97fcc7723577c051bca2aa0831)
 - [aae14cce413b7acc7e445bc874974e76ab6e207e](https://github.com/magento/magento2/commit/aae14cce413b7acc7e445bc874974e76ab6e207e)
 - [86a72f468d06dc21644505667dd0b825f99eb41a](https://github.com/magento/magento2/commit/86a72f468d06dc21644505667dd0b825f99eb41a)
 - [b25aafc59da49ade9bd2855f65d5ec0f13101b79](https://github.com/magento/magento2/commit/b25aafc59da49ade9bd2855f65d5ec0f13101b79)

The fix will be available with the upcoming 2.4.1 release.
